From 332d1a8cf9347404e5f62bf8f674425c51ddb7c9 Mon Sep 17 00:00:00 2001
From: Katsuya Matsubara <matsu@igel.co.jp>
Date: Thu, 5 Jul 2012 10:25:18 +0900
Subject: [PATCH 37/97] Revert "directfb: trick hardware acceleration check"

This reverts commit 4ff79fda9c3ff51497ea0eda3ce1bfdb74dc3a3d.
---
 ext/directfb/dfbvideosink.c |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/ext/directfb/dfbvideosink.c b/ext/directfb/dfbvideosink.c
index f297743..30e081f 100644
--- a/ext/directfb/dfbvideosink.c
+++ b/ext/directfb/dfbvideosink.c
@@ -621,7 +621,7 @@ gst_dfbvideosink_setup (GstDfbVideoSink * dfbvideosink)
         dfbvideosink->hw_scaling = TRUE;
       } else {
         GST_DEBUG_OBJECT (dfbvideosink, "StretchBlit is not accelerated");
-        dfbvideosink->hw_scaling = TRUE;        /* temporary support */
+        dfbvideosink->hw_scaling = FALSE;
       }
 
       dfbvideosink->layer_id = -1;
-- 
1.7.10.4

